CHANGELOG.md

```diff
@@ -2,6 +2,24 @@
 
 Mapbox welcomes participation and contributions from everyone.
 
+# 10.3.0-rc.1 January 26, 2022
+
+## Features ✨ and improvements 🏁
+* Refine viewport plugin's public APIs. ([#1069](https://github.com/mapbox/mapbox-maps-android/pull/1069))
+* Introduce metadata setter API for the legacy offline region. ([#1071](https://github.com/mapbox/mapbox-maps-android/pull/1071))
+* Optimize zooming on terrain and globe. ([#1071](https://github.com/mapbox/mapbox-maps-android/pull/1071))
```

Comment:
just curious - is it actually zoom optimized when indeed using 3D terrain + globe projection? For me it seems more to be optimize zooming when using 3D terrain _or_ globe projection but would be nice to confirm

Reply:
Updated to the change log entry to the original PR, I assume it's for both cases and combined use case as well.